### DISCUSSION TITLE
MX record on google domains

### DISCUSSION THREAD

**Discussion by dev-artisan** [simple-login/app#597] [Q&A]
Hi,
Thanks for providing this wonderful service.
Before committing to premium I was trying out adding one of my domains and got stuck adding the MX records.
The way domains.google.com domains work is that they don't have a field for priority, they make it mandatory to enter the priority followed by a space followed by the record entry, e.g.
10 mx1.simplelogin.co.
The problem with this is when I try to verify on the domains panel in simplelogin, it errors saying:

**_Your DNS is not correctly set. The MX record we obtain is:
10 mx1.simplelogin.co._** 

I wonder if there is a workaround for this other than leaving google domains,
Thanks

**Comment by nguyenkims** [comment]
SimpleLogin actually requires 2 MX records: one for mx1.simplelogin.co and another for mx2.simplelogin.co as a fallback. From the error, it seems that only the first one is set.

**Reply by dev-artisan** [reply]
For some reason I can't mark this as the answer, its glitching for me, I've tried a few browsers now...

**Comment by dev-artisan** [comment]
Awesome thanks for the help, it was an issue with domains.google.com, I didn't know how to add the second record:
![image](https://user-images.githubusercontent.com/1462946/131344631-045d6c52-efee-491e-86bf-6ad84b8819eb.png)

All verified now!

**Reply by nguyenkims** [reply]
Glad that this is solved!

**Reply by rishabhlakhotia** [reply]
On a side note, DNS on Google Domains takes some time to propagate compared to Cloudflare, or NS1. With Cloudflare, it is almost instantaneous, whereas, for Google Domains it takes somewhere between 1-5 minutes. I have 2 domains' DNS on Google DNS, and others on Cloudflare and NS1, this is what I experience.

**Reply by nguyenkims** [reply]
@rishabhlakhotia thanks for the info!
